### PR TITLE
Implement left-top-top navigation design for the responsive_ux layout

### PR DESCRIPTION
**Pull request by dmarcoux**
The `responsive_ux` layout didn't change on small devices, so this explains why previews below are only for large devices.

Before:
before

After:
after

### CODE REVIEW

File: src/api/app/views/layouts/webui/responsive_ux/_places.html.haml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  .navbar-nav
-    .nav.justify-content-end.py-2
-      %button.navbar-toggler{ 'type': 'button', data: { toggle: 'offcanvas' }, aria: { expanded: 'false', label: 'Toggle navigation' } }
-        %i.fas.fa-times
-  %ul.navbar-nav.ml-auto.pt-0.text-nowrap.menu-options
-    %li.nav-item
-      = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
-        = image_tag_for(User.session, size: 23, custom_class: 'rounded-circle bg-light mr-2')
-        Profile
+%h5.pt-2.text-light{ class: navigation == :left ? 'pl-2' : '' }
```
Comment (krauselukas) [inline src/api/app/views/layouts/webui/responsive_ux/_places.html.haml:1]:
```suggestion
%h5.pt-4.text-light{ class: navigation == :left ? 'pl-2' : '' }
```

Comment (krauselukas) [reply src/api/app/views/layouts/webui/responsive_ux/_places.html.haml:1]: A little more padding would create a more clear seperation between the places and actions

Comment (dmarcoux) [reply src/api/app/views/layouts/webui/responsive_ux/_places.html.haml:1]: I agree with you. I've applied your suggestion.

**Comment by krauselukas** [comment]
@dmarcoux the fontsize in the top-navigation are not aligned for every link 

Screenshot_2020-08-13 Profile of Admin(1)

Screenshot_2020-08-13 Profile of Admin

**Comment by krauselukas** [comment]
@dmarcoux besides my last little comment, this looks really good to me, great job! :+1:

**Comment by krauselukas** [comment]
@dmarcoux I found another little thing, the nav-links are cut when too long
Screenshot_2020-08-13 test-package

File: src/api/app/assets/stylesheets/webui/responsive_ux/grid.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+      #left-navigation {
+        position: fixed;
+        top: $top-navigation-height;
+        left: 0;
+
+        & a {
+          color: $gray-300;
+          width: $left-navigation-width;
+          white-space: normal !important;
```
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/webui/responsive_ux/grid.scss:48]: As pointed out by @krauselukas, long links are now wrapping if the text is too long to fit in the available space.  

fix

File: src/api/app/helpers/webui/webui_helper.rb [hunk 3]
```diff
@@ -386,7 +385,7 @@ def link_content(text, css_class, icon)
     if css_class && css_class.include?('nav-link')
       capture do
         concat(tag.i('', class: "fas #{icon}"))
-        concat(tag.div(text, class: 'small'))
+        concat(tag.div(text, class: feature_enabled?(:responsive_ux) ? '' : 'small'))
```
Comment (dmarcoux) [inline src/api/app/helpers/webui/webui_helper.rb:388]: [To fix the font size issue found by @krauselukas](https://github.com/openSUSE/open-build-service/pull/10007#issuecomment-673463363)

**Comment by dmarcoux** [comment]
@krauselukas Please have another look :slightly_smiling_face:

**Comment by dmarcoux** [comment]
> There is a break point between 1200px and 1500px width where project pages are "broken", they overflow to the left.

Yep, I can reproduce this.

**Comment by hennevogel** [comment]
I think it's basically depended on the amount of tabs.

**Comment by dmarcoux** [comment]
> I think it's basically depended on the amount of tabs.

Your hypothesis proved to be right. This is fixed. See changes in `src/api/app/assets/stylesheets/webui/responsive_ux/tabs.scss`.

fix3

**Pull request merged**